Makefile

```diff
@@ -65,4 +75,39 @@ addheaders:
 
 .PHONY: run-e2e
 run-e2e:
-	./hack/run-e2e.sh $(namespace) $(tag) $(native) $(maven_mirror)
\ No newline at end of file
+	./hack/run-e2e.sh $(namespace) $(tag) $(native) $(maven_mirror)
+
+.PHONY: code/build/linux
+code/build/linux:
+	env GOOS=linux GOARCH=amd64 go build $(APP_FILE)
+
+.PHONY: image/build/master
+image/build/master:
+	@echo Building operator with the tag: $(IMAGE_MASTER_TAG)
+	@docker login --username $(REGISTRY_USER) --password $(REGISTRY_PASS) https://registry.redhat.io
+	operator-sdk build $(IMAGE_MASTER_TAG)
+
+.PHONY: image/build/release
+image/build/release:
+	@echo Building operator with the tag: $(IMAGE_RELEASE_TAG)
+	@docker login --username $(REGISTRY_USER) --password $(REGISTRY_PASS) https://registry.redhat.io
+	operator-sdk build $(IMAGE_RELEASE_TAG)
+	operator-sdk build $(IMAGE_LATEST_TAG)
+
+.PHONY: image/push/master
+image/push/master:
+	@echo Pushing operator with tag $(IMAGE_MASTER_TAG) to $(IMAGE_REGISTRY)
```

Comment:
IMAGE_REGISTRY is used here as parameter, but it is not used in commands below, push is hardcoded to quay.io